src/main.rs

```diff
@@ -176,63 +74,6 @@ async fn main() -> Result<()> {
     Ok(())
 }
 
-#[derive(serde::Deserialize, Debug)]
-struct Config {
-    rpc: rpc::Config,
-}
-
-impl Config {
-    fn from_file(f: File) -> Result<Config> {
-        use std::io::{BufReader, Read};
-        let mut reader = BufReader::new(f);
-        let mut buf = Vec::new();
-
-        reader.read_to_end(&mut buf)?;
-        Ok(toml::from_slice(&buf)?)
-    }
-
-    fn from_options(options: &Options) -> Config {
-        Config {
-            rpc: rpc::Config {
-                request_limits: rpc::RequestLimits {
-                    account_info: options.account_info_request_limit,
-                    program_accounts: options.program_accounts_request_limit,
-                },
-                ignore_base58_limit: options.ignore_base58,
-            },
-        }
-    }
-}
-
-use tokio::signal::unix::{signal, SignalKind};
-
-async fn listen_for_subscription_limit(subscriptions_allowed: Arc<AtomicBool>) {
-    let mut stream =
-        signal(SignalKind::user_defined1()).expect("failed to register signal handler for USR1");
-    while stream.recv().await.is_some() {
-        subscriptions_allowed.fetch_xor(true, Ordering::Relaxed);
-    }
-}
-
-async fn config_read_loop(path: PathBuf, rpc: watch::Sender<rpc::Config>) {
```

Comment:
This one shouldn't be removed for compatibility

Reply:
Reverted it back